applications/IgaApplication/custom_python/add_custom_strategies_to_python.h

```diff
@@ -0,0 +1,38 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:		 BSD License
+//					 license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Riccardo Rossi
```

Comment:
check the authors. I always neglect the authors completely, but that is up to you.